include/deal.II/lac/block_matrix_array.h

```diff
@@ -318,6 +318,13 @@ class BlockMatrixArray : public Subscriptor
      * The matrix block itself.
      */
     PointerMatrixBase<typename BlockVectorType::BlockType > *matrix;
+
+    /**
+    * Assignment operator. Since the copy constructor is destructive (see its
+    * documentation) and only exists for convenience there is no reasonable way
+    * to implement this.
+    */
+    Entry &operator= (const Entry &) = delete;
```

Comment:
can you make explicit in the member doc that the function is deleted?

Reply:
Done.